kyuubi-server/src/main/scala/org/apache/kyuubi/engine/JpsApplicationOperation.scala

```diff
@@ -49,12 +49,8 @@ class JpsApplicationOperation extends ApplicationOperation {
     if (runner == null) {
       None
     } else {
-      val pb = "jps -ml" #| s"grep $tag"
-      try {
```

Comment:
shall we keep it?

Reply:
ok

Reply:
According to the comments, it seems not needed.

<img width="687" alt="image" src="https://user-images.githubusercontent.com/6757692/168768500-d4b4fea1-46b4-4e8f-9368-28ac18d7d7dc.png">

Reply:
don't trust it

Reply:
done